lib/app/utils/hl7_validator.rb

```diff
@@ -30,6 +30,15 @@ def validate(resource, fhir_models_klass, profile_url = nil)
       }
     end
 
+    # @return [String] the version of the validator currently being used
+    def version
+      Inferno.logger.info('Fetching validator version')
+      Inferno.logger.info("GET #{@validator_url}/version")
+
+      result = RestClient.get "#{@validator_url}/version"
```

Comment:
The only thing that I think this is missing is fallback code in case the validator isn't running, and/or it doesn't respond to `/version`